osler/surveys/views.py

```diff
@@ -78,19 +78,15 @@ def response(request, id):
 
 def fill(request, pid, id):
     survey = Survey.objects.get(id=id)
+    patient = get_object_or_404(Patient, id=pid)
     ctx = {'survey': survey,
            'QuestionType': Question.QuestionType,
-           'pid': pid
+           'patient': patient
            }
 
-    patient = get_object_or_404(Patient, id=pid)
-
-    active_role = get_active_role(request)
-    can_activate = patient.group_can_activate(active_role)
-
-    if not patient.get_status().is_active and not can_activate:
+    if not patient.get_status().is_active:
         messages.add_message(request, messages.ERROR,
-                             'You are trying to survey an inactive patient, but do not have permission to activate this patient')
```

Comment:
This changes the logic of what happens when you try to survey an inactive patient
**before**: the patient was activated if user has permission to activate patients, otherwise they get an error
**now**: user gets error and is instructed to activate the patient, regardless of their permission level

I know we talked about what made more sense to do @wwick  and @justinrporter but I wanted to make sure this is now correct.

Reply:
I think this is reasonable. Alternatively we could set up some type of setting that could configure how this situation is handled. I think once we get this feature live, Eshwar and UMKC folks may have a strong preference and we can revise it in the future!